gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfigurationPropertyTest.java

```diff
@@ -0,0 +1,118 @@
+package com.google.cloud.hadoop.fs.gcs;
+import com.google.cloud.hadoop.gcsio.GoogleCloudStorageFileSystemOptions;
+import com.google.cloud.hadoop.util.EntriesCredentialConfiguration;
+import com.google.common.collect.ImmutableList;
+import org.apache.hadoop.conf.Configuration;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.util.Collection;
+
+import static com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystemTestBase.loadConfig;
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertThrows;
+
+@RunWith(JUnit4.class)
+public class GoogleHadoopFileSystemConfigurationPropertyTest {
+
+    @Test
+    public void testPropertyCreation() {
```

Comment:
I don't think there big value in this test, because property creation is covered by almost all other tests in this class.

Reply:
ok, thanks however creating a property with nulled deprecated key improves coverage, so I will name method accordingly